scio-core/src/main/scala/com/spotify/scio/coders/KryoAtomicCoder.scala

```diff
@@ -85,6 +86,43 @@ private object KryoRegistrarLoader {
 
 }
 
+
+/** serializers we've written in Scio and want to add to Kryo serialization
+  * @see com.spotify.scio.coders.serializers */
+private object ScioKryoRegistrar extends IKryoRegistrar {
```

Comment:
@anne-decusatis can we make this a `class`? I prefer to have the same semantic as in the `AlgebebirdRegistrar`.

`new ScioRegistrar().apply(kryo)`